geopandas/tools/crs.py

```diff
@@ -0,0 +1,38 @@
+import pyproj
+import re
+import os
+import fiona.crs
+
+
+def explicit_crs_from_epsg(crs=None, epsg=None):
+    """
+    Gets full/explicit CRS from EPSG code provided.
+
+    Parameters
+    ----------
+     crs : dict or string, default None
+         An existing crs dict or string with the 'init' key specifying an EPSG code
+     epsg : string or int, default None
+        The EPSG code to lookup
+    """
+
+    if epsg is None:
+        if crs is not None:
+            if isinstance(crs, str):
+                crs = fiona.crs.from_string(crs)
+            if 'init' in crs and crs['init'].lower().startswith('epsg:'):
+                epsg = crs['init'].split(':')[1]
+
+    if epsg is None:
+        raise ValueError('No epsg code provided.')
+
+    with open(os.path.join(pyproj.pyproj_datadir, 'epsg')) as f:
+        data = f.read()
```

Comment:
I take it this is the only avenue by which `pyproj` provides access to its listing of EPSG codes?